contracts/AnchorVault.vy

```diff
@@ -226,21 +172,19 @@ def petrify_impl():
 
 
 @external
-def emergency_stop():
+def pause():
     """
-    @dev Performs emergency stop of the contract. Can only be called
-    by the current emergency admin or by the current admin.
+    @dev Stops the operations of the contract. Can only be called
```

Comment:
:warning: I am afraid that it Can only be called by the **Lido DAO governance contract**.

Minor proposal:

Actually, we have an ambiguity here with:
```vyper
@internal
def _assert_admin(addr: address):
    assert addr == self.admin # dev: unauthorized


@internal
def _assert_dao_governance(addr: address):
    assert addr == LIDO_DAO_AGENT # dev: unauthorized
```
I would remove one of them tbh (`_assert_dao_governance` would be easier to wipe out)

Reply:
done